broker-core/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -136,7 +136,7 @@ private ActorScheduler initScheduler(final ActorClock clock, final String broker
     final int cpuThreads = cfg.getCpuThreadCount();
     final int ioThreads = cfg.getIoThreadCount();
 
-    Loggers.SYSTEM_LOGGER.info(
+    Loggers.SYSTEM_LOGGER.debug(
```

Comment:
Please remove this log message, we already log the full configuration at the beginning. thanks